resources/generate_book.py

```diff
@@ -576,22 +576,26 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             image('tfc:textures/gui/book/tutorial/wattle_add_stick.png', text_contents='Adding sticks to wattle.'),
             image('tfc:textures/gui/book/tutorial/wattle_add_daub.png', text_contents='Using $(thing)Daub on $(thing)Woven Wattle$() creates a solid block.'),
             image('tfc:textures/gui/book/tutorial/wattle_stained.png', text_contents='It can then be $(thing)stained$() by using dye on it.'),
+            # Clay Blocks and Peat
+            page_break(),
+            # todo: Clay block recipe and peat spawning details/screenshot
+            text('Clay obtained from the ground can be crafted into clay blocks. While unattractive, they are easy to get. When placed and dug again, they\'ll turn back into clay. Peat spawns in the world in patches along water bodies, and is also easy to obtain with stone tools. Peat is flammable, however.', title='Clay Blocks and Peat').anchor('clay_and_peat'),
```

Comment:
This will error when generating, because you've broken the page boundaries. `page_break()` does not *insert* a page break, it requires that the page boundaries that naturally occur, match up with where you expect them to be. Either write this page that you've `//todo`'d, or remove this entry, because it's broken as is.